src/main/java/org/folio/dao/rollover/LedgerFiscalYearRolloverDAO.java

```diff
@@ -46,4 +45,20 @@ public Future<Void> delete(String id, DBClient client) {
     return promise.future();
   }
 
+  public Future<Boolean> validationOfUniqueness(String sql, DBClient client) {
```

Comment:
basically methods should be named as verbs

Reply:
Fixed. Method renamed to isRolloverExists